src/controller/editingcontroller.js

```diff
@@ -23,6 +23,7 @@ import {
 } from '../conversion/model-selection-to-view-converters';
 
 import EmitterMixin from '@ckeditor/ckeditor5-utils/src/emittermixin';
+import '../../theme/theme.scss';
```

Comment:
I know that we used to have global `theme.scss` per repository, but since we use Webpack, we could load styles more locally. This is a perfect example of it. We could have `placeholder.scss` and import it from `placeholder.js`. Thanks to these styles will not be added to the build as long as someone will not include the `js` file (there might be no usage of the `placeholder.js` and, what is more important, it's very simple to find styles used in that JS fine. You just go to import like you would do with a function. //cc @oleq @Reinmar